src/vcpkg/vcpkgcmdarguments.cpp

```diff
@@ -645,15 +645,7 @@ namespace vcpkg
         table.format("", "(default: " + format_environment_variable("VCPKG_DOWNLOADS") + ')');
         table.format(opt(VCPKG_ROOT_DIR_ARG, "=", "<path>"), "Specify the vcpkg root directory");
         table.format("", "(default: " + format_environment_variable("VCPKG_ROOT") + ')');
-        table.format(opt(BUILDTREES_ROOT_DIR_ARG, "=", "<path>"),
-                     "(Experimental) Specify the buildtrees root directory");
         table.format(opt(INSTALL_ROOT_DIR_ARG, "=", "<path>"), "(Experimental) Specify the install root directory");
-        table.format(opt(PACKAGES_ROOT_DIR_ARG, "=", "<path>"), "(Experimental) Specify the packages root directory");
```

Comment:
>Don't remove the buildtrees and packages option from the help. 

They should get removed from help because they are likely to get removed outright in favor of, for example, a single non x- option specifying a place for both. We are aware of the long paths issue (and the 'vcpkg is in a user directory with non-ascii characters' issue) so we won't be breaking scenarios for that, but we don't want to advertise a switch we never intend to make contractual.

>Is there a command to show all available commands of vcpkg? If not there should definitely be one.

No, because there are commands we don't want people to rely on.

Reply:
I put these back for now so that I could merge this but I still think their days are numbered...